Configuration/DataProcessing/python/Impl/ppEra_Run2_2017_trackingOnly.py

```diff
@@ -9,13 +9,13 @@
 import os
 import sys
 
-from Configuration.DataProcessing.Reco import Reco
+from   Configuration.DataProcessing.Impl.trackingOnly import trackingOnly
 import FWCore.ParameterSet.Config as cms
-from Configuration.Eras.Era_Run2_2017_cff import Run2_2017
+from   Configuration.Eras.Era_Run2_2017_cff import Run2_2017
 
-from Configuration.DataProcessing.Impl.pp import pp
+from   Configuration.DataProcessing.Impl.pp import pp
 
-class ppEra_Run2_2017_trackingOnly(pp):
+class ppEra_Run2_2017_trackingOnly(trackingOnly):
     def __init__(self):
         pp.__init__(self)
         # tracking only RECO is sufficient, to run high performance BS at PCL;
```

Comment:
`trackingOnly.__init__(self)` --> done

hen come from the base "automatically": correct. However, in several scenario implementations, inherited data members are initialized, for clarity I guess. I would leave as is.